noir-projects/aztec-nr/aztec/src/encrypted_logs/incoming_body.nr

```diff
@@ -99,6 +76,28 @@ mod test {
         fn deserialize_content(fields: [Field; ADDRESS_NOTE_LEN]) -> Self {
             AddressNote { address: AztecAddress::from_field(fields[0]), owner: AztecAddress::from_field(fields[1]), randomness: fields[2], header: NoteHeader::empty() }
         }
+
+        fn to_be_bytes(self, storage_slot: Field) -> [u8; ADDRESS_NOTE_BYTES_LEN] {
+            let serialized_note = self.serialize_content();
+
+            let mut buffer: [u8; ADDRESS_NOTE_BYTES_LEN] = [0; ADDRESS_NOTE_BYTES_LEN];
+
+            let storage_slot_bytes = storage_slot.to_be_bytes(32);
+            let note_type_id_bytes = AddressNote::get_note_type_id().to_be_bytes(32);
+
+            for i in 0..32 {
+                buffer[i] = storage_slot_bytes[i];
```

Comment:
As mentioned above, we end up overwriting the same bytes multiple times with the same value 👀